content/packages.md

```diff
@@ -93,17 +110,15 @@ There are a handful of other packages that are not in the tidyverse, but are tid
 
 ## Wrangle
 
-As well as [tidyr](http://tidyr.tidyverse.org), and [dplyr](http://dplyr.tidyverse.org), there are five packages designed to work with specific types of data:
+In addition to [tidyr](http://tidyr.tidyverse.org), [dplyr](http://dplyr.tidyverse.org), [stringr](http://stringr.tidyverse.org/), and [forcats](http://forcats.tidyverse.org/), there are three more packages designed to work with specific types of data:
```

Comment:
Hmmmm, this doesn't read quite right. I don't know quite how to put it, but we need to draw the distinction between dplyr and tidyr which work at the data frame level, and all the others which work with individual vectors.

Reply:
I agree, I think it needs a line/fragment explaining the rationale for stringr and forcats core-inclusion for a proper transition. 
~ Some sort of answer to this question:
https://community.rstudio.com/t/what-is-the-core-tidyverse-packages/3628/3?u=mara